### DISCUSSION TITLE
Adding new domain not working

### DISCUSSION THREAD

**Discussion by dhiraj-r** [modoboa/modoboa#2203] [Q&A]
Today, I installed Modoboa on fresh Ubuntu 20.04, also tried on Debian 10.  The good thing is it installed successfully, but adding domain is not working. More specifically **submit** is not working.

**Comment by pklav** [comment]
I had the same problem when I set the option 'Create a domain administrator' to Yes; the Submit button does not react at all. I am able to create a domain when leaving it at **No**.
Hope it helps.
